### DISCUSSION TITLE
怎么设置评论

### DISCUSSION THREAD

**Discussion by izhaorui** [vuepress-theme-hope/vuepress-theme-hope#3038] [Q&A] [answered]
请问关于这个的文档代码怎么具体设置 https://plugin-comment2.vuejs.press/zh/config/giscus.html

**Comment by Mister-Hope** [comment, marked as answer]
文档该说的都说了

- https://plugin-comment2.vuejs.press/zh/guide/giscus.html
- https://giscus.app/zh-CN

你要说你看不懂，那我的建议就是问别人或者别用。
